src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -1109,49 +1109,57 @@ ov::pass::RoPEFusionQwen::RoPEFusionQwen(int split_output_id) {
 ov::pass::RoPEShareCosSin::RoPEShareCosSin() {
     MATCHER_SCOPE(RoPEShareCosSin);
 
-    std::vector<std::shared_ptr<Node>> inputs = {makePattern(), makePattern()};
-    auto const_inv_freq = makePattern<opset1::Constant>({}, {});
-
-    auto Constant_58774 = makeConst(element::u8, ov::Shape({}), {0});
-    auto Broadcast_58775 = makePattern<opset1::Broadcast>({{1.000000f}, inputs[0], Constant_58774},
-                                                          {{"mode", "numpy"}});  //  tensor_array<f32[?,?,?]>
-    auto expand_Broadcast =
-        makePattern<opset1::Multiply>({const_inv_freq, Broadcast_58775},
-                                      {{"auto_broadcast", "numpy"}});  //  tensor_array<f32[?,128,?]>
-    auto matmul_MatMul =
-        makePattern<opset1::MatMul>({expand_Broadcast, inputs[1]}, {{"transpose_a", false}, {"transpose_b", false}});
-    auto transpose_Transpose = makePattern<opset1::Transpose>({matmul_MatMul, {0, 2, 1}});
-    auto cat_Concat = makePattern<opset1::Concat>({transpose_Transpose, transpose_Transpose}, {{"axis", -1}});
-    auto cos_Cos = makePattern<opset1::Cos>({cat_Concat});
-    auto sin_Sin = makePattern<opset1::Sin>({cat_Concat});
-    auto result = makePattern<opset1::Unsqueeze>({cos_Cos | sin_Sin, 1});
+    std::vector<std::shared_ptr<Node>> inputs = {pattern::any_input(), pattern::any_input()};
 
-    matcher_pass_callback callback = [OV_CAPTURE_CPY_AND_THIS](ov::pass::pattern::Matcher& m) {
+    // Broadcast pattern
+    auto const_broadcast_arg = pattern::wrap_type<op::v0::Constant>(pattern::value_matches("{1.000000f}"));
+    auto const_broadcast_axes = pattern::wrap_type<op::v0::Constant>(pattern::value_matches("{0}"));
+    auto broadcast = pattern::wrap_type<op::v3::Broadcast>({const_broadcast_arg, inputs[0], const_broadcast_axes},
```

Comment:
I think 
opset1::Broadcast 
is v1 Broadcast, not v3
Please add  v1 Broadcast or replace it with BroadcastBase class